### PR TITLE
Merging 1.3.0 into master

### PR DESCRIPTION
**Pull request by emagnier**
Merging the `romcal-es6` into `master` and preparing the `1.3.0` release.
Nothing to review here, but this PR needs an approving to be merge.

### CODE REVIEW

File: HISTORY.md [hunk 1]
```diff
@@ -1,5 +1,80 @@
 # Version history
 
+## 1.3.0-rc.1 /2020-12-01
```
Comment (tukusejssirs) [inline HISTORY.md]: I suggest to add `v1.3.0 [final]` release notes. :)

Comment (emagnier) [reply HISTORY.md]: I prefer to do first a release candidate. If everything is going well, then we will release the final version.

**Comment by emagnier** [comment]
It's not the place to do code review, if you want to improve the readme, we still can do it in a new PR.

**Comment by tukusejssirs** [comment]
> It's not the place to do code review, if you want to improve the readme, we still can do it in a new PR.

If I open a new PR, will you merge it **_before_** releasing `v1.3.0`?

**Comment by emagnier** [comment]
We can still do minor improvements, and particularly improve the doc, here during the RC phase.
But I need to release the rc asap, so I can perform tests on several projects that require the romcal module.
When everything gonna be ready for a final release, we will do it at this time.

**Pull request merged**